components/display/nextion.rst

```diff
@@ -60,6 +60,9 @@ Configuration variables:
 - **wake_up_page** (*Optional*, int): Sets the page to display after waking up
 - **auto_wake_on_touch** (*Optional*, boolean): Sets if Nextion should auto-wake from sleep when touch press occurs.
 - **exit_reparse_on_start** (*Optional*, boolean): Request the Nextion exit Active Reparse Mode before setup of the display. Defaults to ``false``.
+- **skip_connection_handshake** (*Optional*, boolean): Sets whether the Nextion display should skip the connection handshake process.
```

Comment:
I think I would rephrase this so it's more correct -- since the _Nextion_ doesn't skip the handshake, but rather _ESPHome_ skips its handshake. haha

```suggestion
- **skip_connection_handshake** (*Optional*, boolean): Sets whether the initial display connection handshake process is skipped.
```